src/acceptanceTest/resources/application-acceptance.yaml

```diff
@@ -14,6 +14,7 @@ sessionsRootContext: /hmi/sessions
 sessions_idRootContext: /hmi/sessions/%s
 listingsRootContext: /hmi/listings
 listings_idRootContext: /hmi/listings/%s
+videoHearingApiRootContext: /hmi/resources/video-hearing
```

Comment:
Remove the Api in the Key value and make the casing small along with plurals....